front/app/hooks/useDevice.ts

```diff
@@ -0,0 +1,15 @@
+import { useBreakpoint } from '@citizenlab/cl2-component-library';
+import { TDevice } from 'components/admin/SelectPreviewDevice';
+
+export default function useDevice() {
+  const isPhone = useBreakpoint('phone');
+  const isTablet = useBreakpoint('tablet');
+
+  const device: TDevice = (() => {
+    if (isPhone) return 'phone';
+    if (isTablet) return 'tablet';
+    return 'desktop';
+  })();
+
+  return device;
+}
```

Comment:
@IvaKop Hey Iva, any thoughts on this one to convert a breakpoint to a device type?

Reply:
If the intention is to reuse this hook everywhere and you find it useful - it makes sense to add it main hook folder. 👍 
Some thoughts:
- As discussed during the last CDM, it's good to have unit tests for everything in our top level folders that are shared by everyone. Here's [what we use](https://react-hooks-testing-library.com/) to test hooks. Also, let's make sure we let the team knows about it afterwards.
- We have one additional breakpoint that you might want to include here - `small-desktop`
- `TDevice` should be collocated with this hook, perhaps, and exported from here instead of it living in a different file?